.github/workflows/onPremPackage.yml

```diff
@@ -0,0 +1,43 @@
+name: On Prem Package
+on:
+  workflow_dispatch:
+    inputs:
+      binaryVersion:
+        type: string
+        description: the version of the tabnine binary which will be packaged into this plugin
+        required: true
+      plugInVersion:
+        type: string
+        description: A valid semver format x.y.z
+        required: true
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - uses: "google-github-actions/auth@v1"
+        with:
+          credentials_json: ${{ secrets.GCS_RELEASE_KEY }}
+      - uses: 'google-github-actions/setup-gcloud@v1'
+      - name: Build
+        id: build
+        run: |
+         echo "downloading binaries"
```

Comment:
do the `sed` thingy?

Reply:
I need to test it. no point changing anything yet